sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -3,12 +3,8 @@
 ## 12.18.0-beta.2 (Unreleased)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Introducing a new field isMultipleClosedAllowed for BlobOutputStream and BlockBlobOutputStream, which allows users to
```

Comment:
We don't have this `isMultipleClosedAllowed` boolean anymore. So, the changelog should be updated before release.

Reply:
Thank you for pointing this out! Let me make the change in a separate PR.